app/models/work.rb

```diff
@@ -18,9 +18,6 @@ class Work < ApplicationRecord
   has_many :events, as: :eventable, dependent: :destroy
 
   validates :state, presence: true
-  validates :license, presence: true, inclusion: { in: License.license_list }
-  validates :subtype, work_subtype: true
-  validates :work_type, presence: true, work_type: true
```

Comment:
We already have these validations in the form, which is more consistent with our validation philosophy. No need to have them here.

Reply:
I think it's a good idea to have them in both places. We want to ensure the licences in the model is SELECTABLE_LICENSES or DISPLAYABLE_LICESES, but we want those in the form to only be the SELECTABLE_LICENSES.  We want to make sure we aren't importing any invalid data (which bypasses the form).

Reply:
@jcoyne I'm open to making this change and it's easy enough. This leaves me fuzzy on the following question: how do we decide where a validation ought to live? What's the guideline?

I hear you regarding not wanting bogus licenses in the database. If that's true, doesn't that imply we want a bunch more of our validations down inthe models? (Less important sidebar: And if they move to the models, would we even need them in the forms anymore?) It'd be great to get a shared sense of this, so I'm copying @sul-dlss/infrastructure-team

Reply:
I am interested in this question too, having previously moved validations around between forms and models.  Our use of draft vs regular forms to deal with validations needed sometimes but not other times just adds to the complexity (and maybe the original driver behind moving most if not all validation outside the model).

Reply:
I think you want to validate things you put in the database (for migrations, etc).  You also want to validate some of the same things on the form (and possibly some more things).  I don't think there is a simple rule of thumb for this. Validations are complex and context dependent.

Reply:
Yeah, this is rather muddy for me, still. I'm happy to embrace the muddiness and rejigger this, though.

Reply:
Done.